### PR TITLE
Add to DG and fix bugs

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: docs/images/vis/radarOutput.png
Comment (NeoMinWei) [inline docs/images/vis/radarOutput.png:1]: nice images

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -562,14 +563,22 @@ Displaying piechart for expense
...
 ![](images/vis/barOuput.png)
 
+Example of usage: `vis /t income /c radar`
+
+```
+Displaying radarchart for income
+```
+
+![](images/vis/radarOutput.png)
+
```
Comment (ryan1604) [inline docs/UserGuide.md:581]: Nice addition

**Review by ryan1604 — APPROVED** [review]
LGTM

**Pull request merged**